token/client/src/token.rs

```diff
@@ -502,6 +502,21 @@ where
         Ok(transaction)
     }
 
+    pub async fn simulate_ixs<S: Signers>(
```

Comment:
Oh dear, I hate that we didn't spell out `instructions` below. I guess we have to stay consistent?
I also dislike `construct_tx`, incidentally.